src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
```

Comment:
Using the universal selector '*' to set margin to 0 can affect all elements globally, which might not be desirable. Consider using more specific selectors to avoid unintended side effects.